schemas/v3.1/meta-openapi.json

```diff
@@ -0,0 +1,113 @@
+{
+  "$schema": "http://json-schema.org/draft/2019-WIP/schema#",
+  "$id": "https://spec.openapis.org/oas/3.1/meta",
+  "$vocabulary": {
+    "https://specs.openapis.org/oas/3.1/vocab/openapi": true
+  },
+  "$recursiveAnchor": true,
+  "title": "OpenAPI Vocabulary Schema",
+  "type": ["object", "boolean"],
+  "allOf": [
+    {
+      "$ref": "http://json-schema.org/draft/2019-WIP/schema"
+    },
+    {
+      "properties": {
+        "type": {
```

Comment:
You don't need to redefine this, as far as I can tell.

Reply:
Greaat! off of memory I thought openapi was more restrictive but the list matches JSON Schema, so away it goes!